app/common/message_moving.py

```diff
@@ -38,11 +38,7 @@
 
 
 def get_ghostty_guild() -> dc.Guild:
-    try:
-        return next(g for g in bot.guilds if "ghostty" in g.name.casefold())
-    except StopIteration:
-        msg = "bot guild name does not contain 'ghostty'"
-        raise ValueError(msg) from None
+    return bot.get_guild(config.guild_id) or bot.guilds[0]
```

Comment:
I think there should be a warning logged.

```suggestion
    if guild := bot.get_guild(config.guild_id):
        return guild
    logger.warning("guild {} not found; defaulting to first guild", config.guild_id)
    return bot.guilds[0]
```

Reply:
If anything, this should be a debug/info log. It's completely fine not to use the env var if the bot is only in one server.